targets/chrome-common

```diff
@@ -31,26 +31,52 @@ REQUIRES='x11'
 . "${TARGETSDIR:="$PWD"}/common"
 
 ### Append to prepare.sh:
-# Only download if it's not already installed, since it auto-updates
 pkg="google-chrome-$CHANNEL"
-if [ -n "`list_uninstalled '' "$pkg"`" ]; then
-    # Chrome needs hicolor-icon-theme to configure/install
-    install hicolor-icon-theme
-
-    url="https://dl.google.com/linux/direct/${pkg}_current_$ARCH.$PKGEXT"
-    tmp="`mktemp crouton.XXXXXX --tmpdir=/tmp`"
-    addtrap "rm -f '$tmp'"
-
-    wget "$url" -O "$tmp"
-
-    # Remove other providers of google-chrome, then install the package
-    for c in stable beta dev; do
-        c="google-chrome-$c"
-        if [ -z "`list_uninstalled '' "$c"`" ]; then
-            remove "$c"
-        fi
-    done
-    install_pkg "$tmp"
+
+# Chrome needs hicolor-icon-theme to configure/install
+install hicolor-icon-theme
+
+# Use the package manager for gentoo, it maintains chrome as well
+if [ "${DISTROAKA:-"$DISTRO"}" = 'gentoo' ]; then
+    # Rename google-chrome-stable to google chrome to match the ebuild
+    if [ "$CHANNEL" = 'stable' ]; then
+        GENTOO_PKG="www-client/google-chrome"
+    else
+        GENTOO_PKG="www-client/$pkg"
+    fi
+
+    # Keyword Chrome
+    keyword_package "$GENTOO_PKG" "$pkg"
+
+    # Accept the license
+    accept_license "$GENTOO_PKG" google-chrome "$pkg"
```

Comment:
I'm mildly uncomfortable with this. How does Gentoo deal with this?

On Chromium OS, build_packages has an --accept-license flag.